local-tests/setup/tinny-environment.ts

```diff
@@ -447,8 +446,9 @@ export class TinnyEnvironment {
    * @throws If there is an error sending the funds.
    */
   getFunds = async (walletAddress: string, amount = '0.001') => {
+    const privateKey = await this.getAvailablePrivateKey();
```

Comment:
I needed the privateKey in scope in the finally{} block, so if I move it inside the try{}, it won't be in scope in the finally{} due to block scoping rules.  also, if this call throws an error, I don't want the finally() to run that tries to release the private key -- since we won't know what private key to release if this call throws an error